### DISCUSSION TITLE
Password protected PDF Popup $i18n placeholders not set

### DISCUSSION THREAD

**Discussion by Djutza** [cefsharp/CefSharp#3398] [Q&A] [answered]
Hi everyone,

we started using CefSharp as PDF viewer in our WPF application. When I try to open Password protected PDF it opens popup as it should but the Resources are missing. If I open same file in google Chrome browser, placeholders are localized.
Problem can be reproduced with MinimalExample, just set Address to password protected  PDF.
Any hints for extending some CEF Handler or javascript?
Cheers!

Chrome:
![Chrome](https://user-images.githubusercontent.com/6997161/107330409-cca43000-6ab1-11eb-8493-9847e22eb63c.PNG)

MinimalExample:
![MinimalWPF](https://user-images.githubusercontent.com/6997161/107330411-cd3cc680-6ab1-11eb-9e6a-940f16e8150c.PNG)

**Comment by amaitland** [comment, marked as answer]
This is a known issue https://bitbucket.org/chromiumembedded/cef/issues/3081/chromium-85-pdf-viewer-cannot-handle-i18n

It should be fixed in version 88. Subscribe to release only notifications here on GitHub to be notified when a new version is available.

**Reply by amaitland** [reply]
You can try out the `88.2.40-pre` release, see https://github.com/cefsharp/CefSharp/issues/3389#issuecomment-776506045

If you still see a problem then you'll need to report it on https://bitbucket.org/chromiumembedded/cef/issues/3081/chromium-85-pdf-viewer-cannot-handle-i18n

**Reply by Djutza** [reply]
It works in `88.2.40-pre` 
Thanks!
